coalib/bears/Bear.py

```diff
@@ -249,8 +249,14 @@ def execute(self, *args, **kwargs):
             result = self.run_bear_from_section(args, kwargs)
             return [] if result is None else list(result)
         except (Exception, SystemExit):
-            self.warn('Bear {} failed to run. Take a look at debug messages'
-                      ' (`-V`) for further information.'.format(name))
+            if self.kind():
```

Comment:
This is incorrect, it'll be true for both local and global bears as well as linter bears making the else unreachable. Instead check it against the constants in the `coalib/bears/BEAR_KIND`.

Also add tests